docs/circuit_diagrams/module_TURRET/VertandHoriz.ino

```diff
@@ -0,0 +1,53 @@
+#include <Servo.h>
+
+Servo vert,hori;  // create servo object to control a servo
+
+int pos = 0;    // variable to store the servo position
+int laser = 12;
+int buzzer = 11;
```

Comment:
We can use existing code to use objects for these, even though this is ultimately what's going on under the hood.